File: python/src/hello_world.py
def print_welcome_message():
    # This variable is interpolated by
    # Boilermaker at build time, and its value is taken from the `boilermaker.toml` file.
    interpolated_message = "{{welcome_message}}";
    print("{interpolated_message}");

def main():
    print("Hello boiled Python!");
    print_welcome_message();

if __name__ == "__main__":
    main()

File: javascript/src/hello_world.js
(() => {
  const print_welcome_message = () => {
      // This variable is interpolated by
      // Boilermaker at build time, and its value is taken from the `boilermaker.toml` file.
      interpolatedMessage = "{{welcome_message}}";
      console.log(`${interpolatedMessage}`);
  }

  console.log("Hello boiled Javascript!");
  print_welcome_message();
})();
